response_operations_ui/views/messages.py

```diff
@@ -191,6 +191,8 @@ def view_conversation(thread_id):
             {"title": "Unavailable"}
         ]
 
+    if refined_thread[-1]['unread']:
```

Comment:
Could this ever throw an IndexError?

Reply:
No, at this point the thread is refined and potential missing values have already been handled